### PR TITLE
Add %{dephell_gensetup} macro instead of the full call to dephell command

### PR DESCRIPTION
**Pull request by mcepl**

### CODE REVIEW

File: spec_cleaner/rpmprep.py [hunk 1]
```diff
@@ -36,6 +36,9 @@ def _cleanup_setup(self, line: str) -> str:
             line = self.strip_useless_spaces(line)
             line = line.replace('%setup', '%setup -q')
 
+        if 'dephell deps convert' in line:
+            line = '%{dephell_gensetup}'
```
Comment (scarabeusiv) [inline spec_cleaner/rpmprep.py]: no curly brackets here.

File: tests/in/python-dephell_changelogs.spec [hunk 2]
```diff
@@ -0,0 +1,70 @@
+#
```
Comment (scarabeusiv) [inline tests/in/python-dephell_changelogs.spec]: Do you really need the whole specfile here?
You can just create file with artifical %prep lines to test it...

**Comment by mcepl** [comment]
Failing tests are because of the difference between ``%{dephell_gensetup}`` and ``%dephell_gensetup``. Not sure which one I should include.

File: spec_cleaner/rpmprep.py [hunk 3]
```diff
@@ -36,6 +36,9 @@ def _cleanup_setup(self, line: str) -> str:
             line = self.strip_useless_spaces(line)
             line = line.replace('%setup', '%setup -q')
 
+        if 'dephell deps convert' in line:
```
Comment (scarabeusiv) [inline spec_cleaner/rpmprep.py]: This should be regexp.

Comment (scarabeusiv) [reply spec_cleaner/rpmprep.py]: We also have 'dephells convert' or at least had before.

Comment (scarabeusiv) [inline spec_cleaner/rpmprep.py]: This should be regex as we have also 'dephells convert'

**Comment by scarabeusiv** [comment]
> Failing tests are because of the difference between `%{dephell_gensetup}` and `%dephell_gensetup`. Not sure which one I should include.

Well as i said in the review it should be without, but you are lacking the %changelog in the outputs.
Also you should maybe also test conversion of other permutations of this command we have in factory.
And last but not least decide wether this replacement really should be done in minimal mode or not, i would say it should not touch anything in minimal.

**Pull request merged**